src/main/java/vn/zalopay/benchmark/util/ExceptionUtils.java

```diff
@@ -0,0 +1,131 @@
+package vn.zalopay.benchmark.util;
+
+import com.alibaba.fastjson.JSONObject;
+
+/**
+ * <b>异常打印工具类</b>
+ * <p>reference: https://github.com/yl-yue/yue-library/blob/j11.2.6.0/yue-library-base/src/main/java/ai/yue/library/base/util/ExceptionUtils.java</p>
+ *
+ * @author	ylyue
+ * @since	2018年9月9日
+ */
+public class ExceptionUtils {
```

Comment:
Please add unit test for this utils class

Reply:
Hello, this is a copied utility class (copied from other open source projects)

Instead of referencing a JAR package, I simply copied the source code for brevity.

Reply:
Hello, in the 'GrpcSamplerTest' use case, the test for ExceptionUtils has been overridden to determine the expected value:

![image](https://user-images.githubusercontent.com/39790401/178677963-5f901e5c-b5af-4346-bb8a-5b19aab07009.png)

Reply:
thanks @yl-yue